app/views/messages/_message.html.erb

```diff
@@ -1,3 +1,6 @@
-<p id="<%= dom_id message %>">
-  <%= message.created_at.to_s(:short) %>: <%= message.content %>
-</p>
+<div id="<%= dom_id message %>">
```

Comment:
When wrapped in a <p>, the button_to form seems to break out of the containing <p>. When wrapped in a <div> the button_to form remains contained in the element. So, changed the <p> to <div> for that reason.